core/src/test/java/org/bitcoinj/wallet/DeterministicKeyChainTest.java

```diff
@@ -750,6 +751,21 @@ public void bloom2() {
             assertTrue("key " + key, filter.contains(key.getPubKeyHash()));
     }
 
+    // TODO: Validate that these strings are ACTUALLY CORRECT ACCORDING TO THE SPECIFICATION
```

Comment:
I just skimmed the documentation for descriptors so please do not assume I got these test strings correct.